prowler/lib/cli/parser.py

```diff
@@ -146,8 +146,8 @@ def __init_outputs_parser__(self):
             "-M",
             nargs="+",
             help="Output modes, by default csv and json-oscf are saved. When using AWS Security Hub integration, json-asff output is also saved.",
-            default=["csv", "json-ocsf"],
-            choices=["csv", "json-asff", "json-ocsf"],
+            default=["csv", "json-ocsf", "html"],
```

Comment:
Please fix the tests related with this change.

Reply:
Done